docs/src/devdocs/interpolations.md

```diff
@@ -1,12 +1,47 @@
 # [Interpolations](@id devdocs-interpolations)
 
-## Type definitions
+All `Interpolation`s should subtype `Interpolation{shape, order}`,
+where `shape <: AbstractRefShape` is the reference shape for which
+the interpolation is defined and [`order`](@ref Ferrite.getorder) is the characteristic interpolation
+order. The [how-to at bottom of this page](@ref devdocs-howto_new-interpolation) describes how to implement a new interpolation.
 
-Interpolations are subtypes of `Interpolation{shape, order}`, i.e. they are
-parametrized by the reference element and its characteristic order.
+## Methods to be implemented for a new interpolation
+### Always
+```@docs
+Ferrite.reference_shape_value(::Interpolation, ::Vec, ::Int)
+Ferrite.reference_coordinates(::Interpolation)
+Ferrite.vertexdof_indices(::Interpolation)
+Ferrite.facedof_indices(::Interpolation)
+Ferrite.facedof_interior_indices(::Interpolation)
+Ferrite.edgedof_indices(::Interpolation)
+Ferrite.edgedof_interior_indices(::Interpolation)
+Ferrite.volumedof_interior_indices(::Interpolation)
+Ferrite.getnbasefunctions(::Interpolation)
+Ferrite.adjust_dofs_during_distribution(::Interpolation)
+```
 
-### Fallback methods applicable for all subtypes of `Interpolation`
+### For special interpolations
+#### Discontinuous interpolations
+For discontinuous interpolations, implementing the following methods might be required.
+```@docs
+Ferrite.is_discontinuous(::Interpolation)
+Ferrite.dirichlet_vertexdof_indices(::Interpolation)
+Ferrite.dirichlet_facedof_indices(::Interpolation)
+Ferrite.dirichlet_edgedof_indices(::Interpolation)
+```
 
+#### Non-identity mapping
+For interpolations that have a non-identity mapping (see
+[Mapping of finite elements](@ref mapping_theory)), the
+mapping type must be specified.
+```@docs
+Ferrite.mapping_type
+Ferrite.get_direction
+```
```

Comment:
We might want to communicate users and devs that these dispatches are only needed if we want to use the FEValues infrastructure. Or do we need them also outside of this?

Reply:
Feels a bit too fine-grained to specify what is needed for different use-cases, e.g. should we also say what is only required if you use the dof distribution? Currently, I don't have anything in mind, but perhaps it might be used outside as well in the future? 

(get_direction mentions that it is used for the mapping in the docstring)